src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -393,7 +393,7 @@ Handle java_lang_String::create_from_symbol(Symbol* symbol, TRAPS) {
 }
 
 // Converts a C string to a Java String based on current encoding
-Handle java_lang_String::create_from_platform_dependent_str(const char* str, TRAPS) {
+Handle java_lang_String::create_from_platform_dependent_str(JavaThread* current, const char* str) {
```

Comment:
This change is incorrect. JNU_NewStringPlatform can raise an exception so TRAPS here is correct.

Reply:
Thanks for finding this.  I reverted this change and the change to as_platform_dependent_str() because it calls GetStringPlatformChars() which calls JNU_GetStringPlatformChars() which can throw an exception.